ios/help/basic/keyboard-usage.md

```diff
@@ -20,7 +20,7 @@ The Gestures:
 
 ![](../ios_images/key-submenu.png)
 
-* **Flick:** hold and slide in various directions on a key to reach alternate outputs. The key will animate to show you the expected output. A common flick is to slide down to reach numerals on the top row of the keyboard.
+* **Flick:** hold and slide in various directions on a key to reach alternate outputs. The key will animate to show you the expected output. For example, a keyboard could support sliding down on keys in the top row to output numerals.
```

Comment:
I'm not following on this one. The word 'may' is present tense conveying the possibility of supporting the flick.

Reply:
As opposed to the original `could`, which is past tense, yes.

Reply:
Oh, oops, I wrote 'could'. 'May' works but 'could' is legit and not past tense in this context. It is commonly used in present tense, as in, "Could I borrow a dollar?" (Especially since 'may' has declined in usage.) Or, "I am so hungry I could eat a horse." Also it's common for subjunctive, as in "If you were three inches shorter, you could be comfortable flying coach." None of those are the past tense of 'can'. If anything, the 'for example' here makes it sound more like the subjunctive. I think this is clear and correct as is, but maybe we could discuss further during a Monday design meeting.

Reply:
Bike shed is out the back fellas. Could would should may must can ... it's not an ISO spec, and I don't think anyone else is going to get stressed reading either of these choices!

For clarity, probably time to park this discussion :grin: